src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -1068,8 +1068,8 @@ CmdTechDrawExtensionDrawCosmCircle3Points::CmdTechDrawExtensionDrawCosmCircle3Po
 {
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Add Cosmetic Circle 3 Points");
-    sToolTipText = QT_TR_NOOP("Add a cosmetic circle based on three vertexes:<br>\
+    sMenuText = QT_TR_NOOP("Add cosmetic circle 3 points");
+    sToolTipText = QT_TR_NOOP("Adds a cosmetic circle based on three vertices:<br>\
```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Adds a cosmetic circle to three selected vertices")
```
then remove next 3 lines